custom/enikshay/integrations/utils.py

```diff
@@ -48,6 +48,18 @@ def is_valid_episode_submission(episode_case):
     return not _is_submission_from_test_location(person_case.case_id, person_case.owner_id)
 
 
+def is_migrated_uatbc_episode(episode_case):
+    try:
+        person_case = get_person_case_from_episode(episode_case.domain, episode_case)
+    except ENikshayCaseNotFound:
+        return False
+    return is_migrated_uatbc_person(person_case)
+
+
+def is_migrated_uatbc_person(person_case):
+    return bool(person_case.get_case_property('migration_created_from_record'))
```

Comment:
for safety I would do ```person_case.get_case_property('migration_created_from_record' == 'true'``` since ```bool('false') == False'```

Reply:
nice catch, thanks, will do!

Reply:
oh wait, nevermind, I remember why I used `bool` - isn't `migration_created_from_record` an ID of a the record in UATBC?